clients/command/src/main/java/org/eclipse/hono/client/command/CommandContext.java

```diff
@@ -117,11 +118,14 @@ default void reject(final String error) {
      *
      * @param tracer The tracer to use.
      * @param command The command for which the span should be started.
-     * @param spanContext Existing span context.
+     * @param spanContext The span context to set as parent span.
+     * @param followsFromSpanContext A span context for which to add a <em>follows-from</em> reference,
```

Comment:
... and this could simply be *followsFromSpan*

Reply:
see above